src/main/java/com/mercadolibre/grupo1/projetointegrador/exceptions/ExceptionCatchStatusCategory.java

```diff
@@ -0,0 +1,10 @@
+package com.mercadolibre.grupo1.projetointegrador.exceptions;
+/*
+@author Gabriel Essenio
+Cria exceçao caso o status de category nao esteja correto
+ */
+public class ExceptionCatchStatusCategory extends RuntimeException{
```

Comment:
Vamo padronizar as exceptions para DESCRICAO + Exception
```suggestion
public class InvalidCategoryException extends RuntimeException{
```

Reply:
Beleza!
Vou mudar a descrição para a padronização